filter.py

```diff
@@ -14,7 +14,10 @@ def line_count(filename):
 
 def filter(annotations, min, max, values):
     for k, v in min.items():
-        if float(annotations.get(k, 0)) < float(v):
+        if k == 'pv':
+            if len(annotations.get(k, 0).split(',')) < int(v):
```

Comment:
The default value of get looks wrong here, considering that the value should be a string.

Reply:
I'm sorry about the bug. Fixed.